src/components/cards/CreatePool/ChooseWeights.vue

```diff
@@ -435,19 +432,22 @@ function onAlertMountChange() {
           </BalAlert>
         </AnimatePresence>
         <AnimatePresence
-          :isVisible="hasInjectedToken && !acceptedCustomTokenDisclaimer"
+          :isVisible="hasUnlistedToken"
           unmountInstantly
           @on-presence="onAlertMountChange"
           @on-exit="onAlertMountChange"
         >
-          <BalAlert :title="$t('tokenWarningTitle')" type="warning">
+          <BalAlert :title="$t('unlistedTokenWarningTitle')" type="warning">
```

Comment:
Please change the type to error because this is a blocking notification.
```suggestion
          <BalAlert :title="$t('unlistedTokenWarningTitle')" type="error">
```